node-runner-cli/templates/radix-fullnode-compose.yml.j2

```diff
@@ -9,6 +9,10 @@ services:
   core:
     cap_add:
     - NET_ADMIN
+    {% if core_node.advanced_user_envs is defined and core_node.advanced_user_envs is not none %}
+    env_file:
```

Comment:
Why not mount the file so that it gets picked up here https://github.com/radixdlt/babylon-node/blob/main/docker/build_scripts/config_radixdlt.sh#L15

That way, the additional config doesn't need a environment variable definition at docker level?  

Or do you think this is better option and why?